tests/glsl/shader-subgroup-arithmetic.slang

```diff
@@ -0,0 +1,99 @@
+// not testing glsl or spirv due to crashing issue from some odd code emitting which looks right but fails
+//TEST:SIMPLE(filecheck=CHECK_GLSL):  -allow-glsl -stage compute -entry computeMain -target glsl
+//TEST:SIMPLE(filecheck=CHECK_SPV):  -allow-glsl -stage compute -entry computeMain -target spirv
+
+// not testing hlsl due to missing impl
+// TEST:SIMPLE(filecheck=CHECK_HLSL): -allow-glsl -stage compute -entry computeMain -target hlsl
```

Comment:
When targeting HLSL, or others if possible, can you narrow down more to which tests are causing problems?
I don't think ALL of arithmetic functions are having problems for HLSL.
If there are some functions working for HLSL, we should test them instead of disabling the entire HLSL test.